.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale pull requests
+
+on:
+  schedule:
+  - cron: '36 5 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+        stale-pr-label: 'no-pr-activity'
```

Comment:
@milljm Sorry for commenting after this has been merged, but should this be `stale` instead, as this is what we used with the last stalebot?

Reply:
yeah probably!

Reply:
Alright, I'll make a quick adjustment if you haven't already

Reply:
See #20521